nautobot/docs/plugins/development.md

```diff
@@ -864,6 +864,77 @@ class RandomAnimalView(View):
 
 This view retrieves a random animal from the database and and passes it as a context variable when rendering a template named `animal.html`, which doesn't exist yet. To create this template, first create a directory named `templates/nautobot_animal_sounds/` within the plugin source directory. (We use the plugin's name as a subdirectory to guard against naming collisions with other plugins.) Then, create a template named `animal.html` as described below.
 
+### Utilizing Nautobot Generic Views
```

Comment:
Broadly this information is not plugin-specific - I'd personally prefer to see it as a new page under `nautobot/docs/development/`, similar to how we have `nautobot/docs/development/templates.md` describing how to use the generic templates; it could then be linked from here as a cross-reference.